src/Pages/ProductListingPage/ProductListingPage.jsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import { FilterSidebar } from "../../Components/Index";
```

Comment:
if you rename `Index` with `index` you don't even need to mention it  in the path.